PWGCF/TwoParticleCorrelations/Tasks/corrSparse.cxx

```diff
@@ -51,10 +51,15 @@ struct CorrSparse {
   {
     LOGF(info, "Starting init");
     registry.add("Yield", "pT vs eta vs Nch", {HistType::kTH3F, {{40, 0, 20, "p_{T}"}, {100, -2, 2, "#eta"}, {100, 0, 100, "Nch"}}}); // check to see total number of tracks
-    registry.add("etaphi_Trigger", "eta vs phi vs Nch", {HistType::kTH3F, {{100, -2, 2, "#eta"}, {200, 0, 2 * M_PI, "#varphi"}, {100, 0, 100, "Nch"}}});
+    registry.add("etaphi_Trigger", "eta vs phi vs Nch", {HistType::kTH3F, {{100, -2, 2, "#eta"}, {200, 0, TwoPI, "#varphi"}, {100, 0, 100, "Nch"}}});
 
+    // Make histograms to check the distributions after cuts
     registry.add("deltaEta_deltaPhi_same", "", {HistType::kTH2D, {axisDeltaPhi, axisDeltaEta}}); // check to see the delta eta and delta phi distribution
     registry.add("deltaEta_deltaPhi_mixed", "", {HistType::kTH2D, {axisDeltaPhi, axisDeltaEta}});
+    registry.add("Phi", "Phi", {HistType::kTH1D, {{72, 0, TwoPI, "Phi"}}});
+    registry.add("Eta", "Eta", {HistType::kTH1D, {{40, -2, 2, "Eta"}}});
+    registry.add("pT", "pT", {HistType::kTH1D, {axisPtTrigger}});
+    registry.add("Nch", "N_{ch}", {HistType::kTH1D, {axisMultiplicity}});
 
     registry.add("Sparse_mixed", "", {HistType::kTHnSparseD, {{axisVertex, axisPtTrigger, axisPtAssoc, axisMultiplicity, axisDeltaPhi, axisDeltaEta}}}); // Make the output sparse
     registry.add("Sparse_same", "", {HistType::kTHnSparseD, {{axisVertex, axisPtTrigger, axisPtAssoc, axisMultiplicity, axisDeltaPhi, axisDeltaEta}}});
```

Comment:
I would recommend two thing in here
- do you really need `double`?
- have a look at the `CorrelationContainer` under `PWGCF/Core` and one of its usages within `PWGCF/Tasks/correlations.cxx` 

This histograms are going to be really big. The `double` to `float` directly half the size
The standard `THnSparse` uses two bins per dimension to store underflow and overflow which with six dimensions this is a significant amount of memory. The `CorrelationContainer` uses an `O2` version of `THnSparse` which does not incorporate the underflow and overflow bins